autoload/vista/finder/fzf.vim

```diff
@@ -86,6 +86,7 @@ function! vista#finder#fzf#sink(line) abort
   call vista#source#GotoWin()
   call cursor(lnum, col)
   normal! zz
+  call call('vista#util#Blink', get(g:, 'vista_top_level_blink', [2, 100]))
```

Comment:
```suggestion
  call call('vista#util#Blink', get(g:, 'vista_blink', [2, 100]))
```

You could be misled by the mistake in `jumps.vim`, `vista#jump#TagLine(tag)` should use `g:vista_blink` too.

Reply:
@liuchengxu 

Thanks 🙏 Fixed!